packages/apps/plugins/plugin-pwa/src/PwaPlugin.tsx

```diff
@@ -26,7 +26,7 @@ export const PwaPlugin = (): PluginDefinition => ({
               id: `${PWA_PLUGIN}/need-refresh`,
               title: translations[0]['en-US'][PWA_PLUGIN]['need refresh label'],
               description: translations[0]['en-US'][PWA_PLUGIN]['need refresh description'],
-              duration: 240e3, // 4 minutes
+              duration: 4 * 60 * 1000, // 4m
```

Comment:
not sure how this is clearer?

Reply:
(personally [I’ve often used this syntax](https://gitlab.com/accord.coop/accord/-/blob/staging/api/lib/accord_slack/views/new_decision_step_2.ex#L91-126), though it’s more valuable when multiple durations are defined together.